src/packages/commit-lint/lib/lib.js

```diff
@@ -0,0 +1,70 @@
+const fs = require('fs');
+const { ArgumentParser } = require('@eigenspace/argument-parser');
+
+function exitWithSuccess(message) {
+    // eslint-disable-next-line no-console
+    console.info(message);
+    process.exit(0);
+}
+
+function exitWithError(message) {
+    // eslint-disable-next-line no-console
+    console.error(message);
+    process.exit(1);
+}
+
+function getCommitMessage() {
+    const parser = new ArgumentParser();
+    const args = parser.get(process.argv);
+
+    const message = args.get('message');
+
+    if (message === 'COMMIT_MESSAGE') {
+        return fs.readFileSync('.git/COMMIT_EDITMSG', 'utf8');
+    }
+
+    return message;
```

Comment:
Yes. And maybe for using not only in git envieroment.

Reply:
It is difficult to image how this tool will be used for another version control system. It seems it can not be, can it?
If this thing is for testing purpose, I suggest to make reading commit message by default without any params and add parameter `--mock=<message>` for testing purpose.